core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -191,9 +191,7 @@ impl PrioGraphScheduler {
                 saturating_add_assign!(num_scheduled, 1);
 
                 let sanitized_transaction_ttl = transaction_state.transition_to_pending();
-                let cu_limit = transaction_state
-                    .transaction_priority_details()
-                    .compute_unit_limit;
+                let cost = transaction_state.transaction_cost().sum();
```

Comment:
Just a quick check: for Vote, its `cu_limit` is always `0`, but has value of `3428` for `cost`. By using different `CU`s, will it impact how vote's scheduled?

Reply:
vote's don't go through the scheduler - they have their own independent threads still.

Reply:
this PR realy has no impact on its' own. The CUs are tracked, but not used for load-balancing currently because they are so unreliable. This is intended to be used later on, but right now its' just tracked, so changing from cu_limit to cost shouldn't impact anything.

Reply:
Gotcha, good to know it is for the purpose of load-balancing, not counting towards the limits. 

Also worth noting that `transaction_cost` includes `requested_cu_limit`, if used. So that unreliability also in `transaction_cost` atm.

Reply:
Right, a major difference here is that for simple txs which we know the cost of, it will not include "requested_cus". For example a simple transfer. So for purposes of prioritizing by reward/cu we want to use this number

Reply:
that would de-prioritize txs with large requested_cu but actually not-so-largge_actual-cu, if signature fee is included in reward/cu calculation, would it?